management.md

```diff
@@ -14,6 +14,25 @@
 - How will you run the course sessions to ensure that you use them
   to maximum effectiveness?
 
+  The first step to ensure that the session is run efficiently will be to
+  have a quick planning meeting with all the team leaders in order to develop
+  an actionable strategy for meeting and achieving the project's requirements.
+  This will involve a clear, detailed breakdown of which team is responsible
```

Comment:
This sentence could use some restructuring. Other than that, all looks fine.

Reply:
I changed the sentence to make it sound better.